lib/foreman_maintain/utils/backup.rb

```diff
@@ -241,6 +241,24 @@ def validate_hostname?
         end
       end
 
+      def validate_interfaces?
```

Comment:
Few thoughts,
1. The invalid interfaces can be dhcp or dns or both. So if user fixes one of the interface maybe in next restore run he will need to fix another one?
2. The method does not return which interface is invalid, or what is expected. I think if we show that, it can help? because we are failing on invalid interfaces which means it needs to be fixed to successfully restore? or skip the check.
3. Should we provide a way to fix it by either suggesting change the name of interface or change the dns or dhcp config on successful restore. This means user can whitelist or skip this check and proceed with restore.

Reply:
Yeah, I do agree. I think I was mostly not sure about the interface/contract (if there is any) between `ForemanMaintain::Utils` and `Checks`. The other checks seemed to use a boolean `validate_XXX?`, which I copied over.
If there is no such hard requirement, I'd probably return a `Hash` of the form `{feature: {'configured': …, 'available': …}, …}` and let the check look whether the hash is empty (no errors) or construct a message from the hash based on features and configured/available.